packages/stark-ui/src/modules/table/entities/column-properties.intf.ts

```diff
@@ -69,6 +74,7 @@ export interface StarkTableColumnProperties {
 	onClickCallback?: (value: any, row?: object, columnName?: string) => void;
 
 	/**
+	 * @deprecated
```

Comment:
Deprecated? Normally we will keep it with the difference that `PERSIST` and `HIDDEN` will not be accepted anymore right?